frontend-react/src/components/ManagePublicKey/ManagePublicKey.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+import { AuthElement } from "../AuthElement";
+import { withCatchAndSuspense } from "../RSErrorBoundary";
+import { MemberType } from "../../hooks/UseOktaMemberships";
+
+export function ManagePublicKey() {
+    return (
+        <div className="grid-container margin-bottom-5 tablet:margin-top-6">
```

Comment:
Let's use paddings instead of margins when we're restricting widths/heights of contents; margins are meant for spacings between elements rather than dictating the descendant content flow, and vertical margins are also susceptible to [collapse](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing), which can happen unintentionally especially between top-level elements of components.

Reply:
Just following the way we have it throughout the site but can change it. Will also change from <div> to <section> per your comment below.